cg/meta/upload/scout/uploadscoutapi.py

```diff
@@ -106,6 +104,17 @@ def add_scout_config_to_hk(
         LOG.info("Added scout load config to housekeeper: %s", config_file_path)
         return file_obj
 
+    def get_multiqc_html_report(self, case_id: str) -> Tuple[str, Optional[hk_models.File]]:
+        """Get a multiqc report for case in housekeeper
+
+        Args:
+            case_id     (string):       Case identifier
+        Returns:
+            multiqc-html-report (housekeeper.store.models.File): Multiqc report
+        """
+        multiqc_tag = "multiqc-html"
+        return (multiqc_tag, self.housekeeper.files(bundle=case_id, tags=[multiqc_tag]).first())
```

Comment:
```suggestion
        return (MULTIQC_HTML_TAG, self.housekeeper.find_file_in_latest_version(case_id=case_id, tags=[MULTIQC_HTML_TAG])
```
Would something like this work? And since the returned MULTIQC_HTML_TAG always is the same do we need to return it or can we just use the constant when invoking in the caller?

Reply:
Yep, lets just use the constants

Reply:
I think it is a good thing, because basically you want he code to croak if this tag changes. And if it changes it is easy to find all occurrences and update them. There are tags in: cg/constants/tags.py